crates/starknet-os/src/starkware_utils/commitment_tree/binary_fact_tree_node.rs

```diff
@@ -174,20 +174,7 @@ where
         ffc: &'a mut FactFetchingContext<S, H>,
         indices: &'a [TreeIndex],
         facts: &'a mut Option<BinaryFactDict>,
-    ) -> impl std::future::Future<Output = Result<HashMap<TreeIndex, LF>, TreeError>> + Send {
```

Comment:
Any reason why you have removed this default implementation?

Reply:
No particular reason except it's never used. This trait is only implemented in `virtual_patricia_node.rs`, and it has it's own implementation of _get_leaves.